src/documentation.js

```diff
@@ -304,6 +304,8 @@ Documentation.Member = class {
     };
     this.async = false;
     this.alias = name;
+    /** @type {'param' | 'option'} */
+    this.argType = null;
```

Comment:
`this.type` vs `this.argType` is confusing. i'd call it differently, `this.paramOrOption` :)